events/search_index/postgres.py

```diff
@@ -49,9 +49,18 @@ def update_search_index(cls, event: Event) -> None:
                     "place_last_modified_time": event.location.last_modified_time
                     if event and event.location
                     else timezone.now(),
-                    "words_fi": list(cls.get_words(event, "fi")),
-                    "words_sv": list(cls.get_words(event, "sv")),
-                    "words_en": list(cls.get_words(event, "en")),
+                    "words_fi_weight_a": list(cls.get_words(event, "fi", "A")),
```

Comment:
We could add the whole set of languages for different weighs like:
```suggestion
                   **{ f"words_{lang}_weight_a": list(cls.get_words(event, lang, "A")) for lang in settings.LANGUAGES},
                   **{ f"words_{lang}_weight_b": list(cls.get_words(event, lang, "B")) for lang in settings.LANGUAGES},
                   **{ f"words_{lang}_weight_c": list(cls.get_words(event, lang, "C")) for lang in settings.LANGUAGES},
                   ...
```
This could be applied to all of below similar words updates  / creates as well.

Also while checking this I stumbled on FULLTEXT_SEARCH_LANGUAGES settings, but not sure if that something we want ot get ridoff or keep 🤔